Retarus Event Collector (#38096)
demisto/content: Packs/RetarusSecureEmailGateway/Integrations/RetarusSecureEmailGateway/RetarusSecureEmailGateway_description.md

> ## BaseIntegration Help

Reviewer: Please correct this.
Notice that this will show as help section, meaning where to get the params from and how to configure the instance.

## Retarus Integration help

> Provide the api token that Retarus has provided to you.

Reviewer 2: ```suggestion
Provide the API token Retarus created for you.
```

> Provide the channel you created or use the default channel.

Reviewer: In Retarus Secure Email Gateway, the channel is typically used to define specific configurations or processing rules for email traffic. Channels allow you to create distinct processing pipelines for different types of email communication, based on various criteria, such as sender, recipient, domain, or other metadata.

Common Uses of Channels:
Custom Processing Rules: Channels can apply specific email filtering, encryption, or other processing rules tailored to particular departments, business units, or workflows.

Routing and Forwarding: Channels can dictate how emails are routed—e.g., ensuring emails to/from certain domains go through a specific route or server.

Compliance Policies: Some channels might enforce compliance with regulations, such as automatically encrypting sensitive emails or applying specific archiving rules.

Quarantine and Filtering: Channels may define which emails are quarantined, blocked, or tagged as spam based on unique filtering criteria.

Audit and Logging: Emails processed via a specific channel may have different logging or auditing rules to meet organizational or regulatory requirements.

By configuring channels, administrators can fine-tune email traffic management to align with the organization's security, compliance, and operational needs.

 ___
 
This is what I get for the channels, consider adding some data about the channels.

Author: I also added in the readme
